### DISCUSSION TITLE
Passing on the zwavejs2mqtt device names to the zwave-js-server

### DISCUSSION THREAD

**Discussion by larstobi** [zwave-js/zwave-js-ui#516] [General]
The MQTT discovery will pass the custom node names on to HASS, and as a bonus it's customizable using a template. However, I tried the ZWave JS integration with HASS, and the names only show up as autogenerated device names.

Is it even possible to pass along the custom names to the ZWave JS server and HASS, so that the names I have set for the nodes in zwavejs2mqtt would be the device and entity names in the ZWave JS integration in HASS?

**Comment by robertsLando** [comment]
I dunno how hass creates the devices entity names, If they use the internal nodes name/location I should write them there instead of my own file `nodes.json`
